frontend/packages/topology/src/behavior/useDndDrop.tsx

```diff
@@ -110,6 +110,11 @@ export const useDndDrop = <
             return false;
           }
 
+          if (nodeRef.current instanceof SVGRectElement) {
+            const { x: bboxx, y: bboxy, width, height } = nodeRef.current.getBBox();
+            if (x < bboxx || x > bboxx + width || y < bboxy || y > bboxy + height) return false;
+          }
```

Comment:
I think we talked about this before. The original impl is incorrect.
That is you only need to do this:
```
const { x: bboxx, y: bboxy, width, height } = nodeRef.current.getBBox();
if (x < bboxx || x > bboxx + width || y < bboxy || y > bboxy + height) return false;
```

You can remove `if (nodeRef.current instanceof SVGRectElement) {` and remove the old impl which was deconstructing `left, right, top, bottom` because that was wrong.